### PR TITLE
Feature/simple slot filler and go_bot database

### PR DESCRIPTION
**Pull request by vikmary**
- [x] modified simple slot filler from aCombray and docs
- [x] add database api calls to DSTC2 dataset -> new dataset `dstc2_v2`
- [x] add SqliteDatabase component + docs
- [x] add confident threshold to slotfiller
- [x] add `intents_dstc2_big.json` that rely on `wiki.en.bin` embeddings
- [x] minor upgrades of `go_bot` (variational dropout and l2 regularization)
- [x] retrain `gobot_dstc2.json` and `gobot_dstc2_best.json`, upload on share and update metrics

### CODE REVIEW

File: deeppavlov/core/data/database.py [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+                if curr_record != record:
+                    to_update[primary_values] = record
+            else:
+                to_insert[primary_values] = record
+
+        if to_insert:
+            fformat = '(' + ','.join(['?'] * len(self.keys)) + ')'
+            self.cursor.executemany("INSERT into {}".format(self.tname) +
+                                    " VALUES {}".format(fformat),
+                                    to_insert.values())
```
Comment (yoptar) [inline deeppavlov/core/data/database.py]: Why not
```
fformat =  ','.join(['?'] * len(self.keys))
self.cursor.executemany("INSERT into {} VALUES ({})".format(self.tname, fformat), to_insert.values())
```
?

Comment (vikmary) [reply deeppavlov/core/data/database.py]: Yeah, this way is defiantly better. I'll fix

File: deeppavlov/core/data/sqlite_database.py [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+    def __call__(self, batch: List[Dict],
+                 order_by: str = None,
+                 ascending: bool = False) -> List[List[Dict]]:
+        order = 'ASC' if ascending else 'DESC'
+        if not self._check_if_table_exists():
+            log.warn("Database is empty, call fit() before using.")
+            return [[] for i in range(len(batch))]
+        elif not self.keys:
+            self.keys = self._get_keys()
```
[the remark this answers is not in the document]
Comment (vikmary) [reply deeppavlov/core/data/sqlite_database.py]: True. Will make the fix

File: deeppavlov/skills/go_bot/network.py [hunk 3]
```diff
@@ -184,10 +210,9 @@ def _add_placeholders(self):
 
     def _build_body(self):
         # input projection
-        _units = tf.nn.dropout(self._features, self._dropout)
-        _units = tf.layers.dense(_units, self.dense_size,
-                                 kernel_initializer=xav(), name='units')
-
+        _units = tf.layers.dense(self._features, self.dense_size,
```
Comment (mu-arkhipov) [inline deeppavlov/skills/go_bot/network.py]: May be some activation function here?

Comment (vikmary) [reply deeppavlov/skills/go_bot/network.py]: Sounds good, but not in the pull-request =)
